src/ipc/utils/eigen_ext.hpp

```diff
@@ -138,6 +138,8 @@ project_to_pd(
     const Eigen::Matrix<_Scalar, _Rows, _Cols, _Options, _MaxRows, _MaxCols>& A,
     double eps = 1e-8);
 
+enum class ProjectType { None, Clamp, Abs };
```

Comment:
Can you rename to `PSDProjectionMethod` to make it more self explainable?

Also, can you use all caps for enum members to match the repo style?